packages/server/test/ui/storybook.test.jsx

```diff
@@ -64,9 +64,12 @@ initStoryshots({
     getMatchOptions: () => ({
       failureThreshold: process.env.CI ? 0.005 : 0.0015,
       failureThresholdType: 'percent',
+      // Slower, but required because images can be larger than the default maxBuffer for child processes
+      // which jest-image-snapshot relies on.
+      // https://github.com/americanexpress/jest-image-snapshot/issues/210
+      runInProcess: true,
     }),
     getScreenshotOptions: () => ({
-      encoding: 'base64',
```

Comment:
I was getting this error:

```
Invalid file signature

      at Object.<anonymous>.module.exports.Object.<anonymous>.Parser._parseSignature (node_modules/pngjs/lib/parser.js:53:18)
      at Object.<anonymous>.module.exports.Object.<anonymous>.SyncReader.process (node_modules/pngjs/lib/sync-reader.js:34:17)
      at Object.<anonymous>.module.exports (node_modules/pngjs/lib/parser-sync.js:68:10)
      at Object.<anonymous>.exports.read (node_modules/pngjs/lib/png-sync.js:10:10)
      at diffImageToSnapshot (node_modules/@storybook/addon-storyshots-puppeteer/node_modules/jest-image-snapshot/src/diff-snapshot.js:119:39)
      at Object.toMatchImageSnapshot (node_modules/@storybook/addon-storyshots-puppeteer/node_modules/jest-image-snapshot/src/index.js:187:7)
      at __EXTERNAL_MATCHER_TRAP__ (node_modules/expect/build/index.js:342:30)
      at Object.throwingMatcher [as toMatchImageSnapshot] (node_modules/expect/build/index.js:343:15)
      at node_modules/@storybook/addon-storyshots-puppeteer/dist/imageSnapshot.js:74:43
      at step (node_modules/@storybook/addon-storyshots-puppeteer/dist/imageSnapshot.js:44:23)
      at Object.next (node_modules/@storybook/addon-storyshots-puppeteer/dist/imageSnapshot.js:25:53)
      at fulfilled (node_modules/@storybook/addon-storyshots-puppeteer/dist/imageSnapshot.js:16:58)
          at runMicrotasks (<anonymous>)
```

I walked through the stack here and it seemed the expectation is that the image data is provided to `jest-image-snapshot` as a Buffer... not sure how this was working before, but I deleted the `encoding: base64` (defaults to binary/Buffer) and the error went away.